packages/authentication/src/keys.ts

```diff
@@ -9,7 +9,7 @@
 export namespace BindingKeys {
   export namespace Authentication {
     export const STRATEGY = 'authentication.strategy';
-    export const PROVIDER = 'authentication.provider';
-    export const METADATA = 'authenticate';
+    export const AUTH_ACTION = 'authentication.actions.authenticate';
+    export const METADATA = 'authentication.operation-metadata';
```

Comment:
In core, we are using `sequence.actions.` prefix for the built-in actions like `sequence.actions.findRoute`.

I was not sure what casing style to use for multi-world names like "operationMetadata". I see that we are using camelCase for `findRoute`, so I'll do the same for `operationMetadata`.